### DISCUSSION TITLE
Error while trying to build simple package with CsWinRT

### DISCUSSION THREAD

**Discussion by bryaan** [microsoft/WindowsAppSDK#2034] [Q&A] [answered]
I install Microsoft.Windows.CsWinRT version 1.5.0-prerelease.220124.4 but I get the following build error:

`  Microsoft.WindowsAppSDK.targets(29, 97): [MSB4086] A numeric comparison was attempted on "%(NuGetPackageVersion)" that evaluates to "1.5.0-prerelease.220124.4" instead of a number, in condition "'%(Filename)%(Extension)'=='WinRT.Runtime.dll' and %(NuGetPackageVersion)>1.3".`

**Comment by BenJKuhn** [comment, marked as answer]
This is an issue in the Win App SDK props target that we're tracking in issues now. There is a workaround available here as well: https://github.com/microsoft/WindowsAppSDK/issues/2042
